server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreBaseIntegTestCase.java

```diff
@@ -169,14 +172,26 @@ protected Settings nodeSettings(int nodeOrdinal) {
                             repoType,
                             REPOSITORY_2_NAME,
                             translogRepoPath,
+                            repoType,
+                            REPOSITORY_3_NAME,
+                            routingTableRepoPath,
```

Comment:
We can keep Routing table related settings limited to RoutingTable IT and not pollute existing tests, and ensure these work without routing table as well.

Reply:
Addressed it in revision.